From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@trinadhk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (trinadhk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

trinadhk commented:
Please do not merge. Committed to wrong branch.

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

hpsin committed:
Fix Azure AD version number

cherylmc committed:
service_support

Commit:
Updates Local Emulator documentation for the 1.21.0.6 release published on 2018-03-27

Commit:
Corrected a grammatical error

squillace committed:
updating the content to 0.12 version

squillace committed:
fixed

squillace committed:
removing brew output. useless, and changes until we get stable.

gsacavdm committed:
Removing bad section in gov-compute

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

MarkMcGeeAtAquent committed:
updates

miaojiang committed:
modified Azure Monitor and AAD docs

miaojiang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

miaojiang committed:
cosmetics changes

squillace committed:
finishing up the document

squillace committed:
last steps and clean up

cherylmc committed:
support

tamram committed:
updating storage account limits

rwike77 committed:
moved security articles in toc

cherylmc committed:
powershellinteractive

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' into ersupport

rwike77 committed:
updated toc

rwike77 committed:
updated intro

rwike77 committed:
fixed typos

v-nagta committed:
USER STORY 1222911 SaaS App Tutorial: Workplace by Facebook update

v-nagta committed:
Licensing #6263 Note add

v-viinde committed:
USER STORY 1223010 SaaS App Tutorial: Amazon Web Services (AWS) Update

v-viinde committed:
Merge branch 'master' into awsuiupdate

Commit:
xregion-peering-ga

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into xregion-peering-ga

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into xregion-peering-ga

v-geberr committed:
ux fix

CarlRabeler committed:
Update sql-database-features.md

CarlRabeler committed:
Update sql-database-features.md

v-geberr committed:
edits

dlepow committed:
N-series: storage fixes

v-geberr committed:
edits

CarlRabeler committed:
Update sql-database-features.md

v-shils committed:
Merge pull request #36719 from v-geberr/0330-screenshot-fix-7

ux fix

CarlRabeler committed:
Update sql-database-features.md

neilpeterson committed:
[AKS] next steps patch

cherylmc committed:
Update expressroute-faqs.md

neilpeterson committed:
fixes MicrosoftDocs/azure-docs#6519

paulmey committed:
Add australiaeast CDS cluster IP

We've deployed australiaeast-cds.australiaeast.cloudapp.azure.com in Australia East. It is not yet operational, but we want to add the IP some time before we add it to the traffic manager profile.

Commit:
ASG GA

PRMerger16 committed:
Merge pull request #36443 from cherylmc/ersupport

service_support

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ASG-GA

dlepow committed:
Batch: metadata upates

PRMerger18 committed:
Merge pull request #36720 from CarlRabeler/patch-39

Update sql-database-features.md

squillace committed:
Update kubernetes-draft.md

rboucher committed:
remove the word preview

dlepow committed:
more

hpsin committed:
Fix claims not in use. 

The compliance claims for SAML tokens are not actually valid. Removing.

PRMerger19 committed:
Merge pull request #36164 from hpsin/patch-1

Fix Azure AD version number

rboucher committed:
remove preview

mikeketchum committed:
Update documentation-government-networking.md

rboucher committed:
remove near real time and newer

PRMerger20 committed:
Merge pull request #36624 from MarkMcGeeAtAquent/table-update-node-1

Cosmos DB: Table API updates for Node.js

KumudD committed:
updated links to lb topics

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into lbaz0330

tfitzmac committed:
removed cloud shell deployment

mikeketchum committed:
Update documentation-government-networking.md

Commit:
Merge pull request #36729 from hpsin/patch-4

Fix claims not in use.

kgremban committed:
Updating author values for iot-edge

Commit:
Merge pull request #36731 from rboucher/patch-2

remove preview

Commit:
Merge pull request #36723 from neilpeterson/aks-quickstart-patch

[AKS] next steps patch

tamram committed:
fixing ZRS issues

roygara committed:
Refreshing articles and fixing bugs.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Refresh

tfitzmac committed:
fixed link

Commit:
Updated accounts to use information

mikeketchum committed:
BGP community update

Added new regions

mikeketchum committed:
Update documentation-government-networking.md

PRMerger10 committed:
Merge pull request #36675 from cherylmc/devices2

powershellinteractive

Commit:
Updated accounts information

KumudD committed:
fixed formatting

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into lbaz0330

tamram committed:
Fixing broken links

PRMerger13 committed:
Merge pull request #36734 from tfitzmac/0330firsttemplate

removed cloud shell deployment

roygara committed:
Fixing links.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Refresh

PRMerger18 committed:
Merge pull request #36735 from kgremban/mar30-edgeauthor

Updating author values for iot-edge

KumudD committed:
toc update

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into lbaz0330

ktoliver committed:
remove "AAD" acronym

remove "AAD" acronym

Commit:
Spaced paras better

v-geberr committed:
web app bot template update

PRMerger19 committed:
Merge pull request #36732 from mikeketchum/patch-1

Update documentation-government-networking.md

v-shils committed:
Merge pull request #36721 from dlepow/fix

N-series: storage fixes

rolyon committed:
Collected role assignments unders one node

v-shils committed:
Merge pull request #36728 from rboucher/patch-1

remove the word preview

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-toctweaks

ktoliver committed:
Merge pull request #36465 from squillace/master

updating the content to 0.12 version

ktoliver committed:
Merge pull request #36730 from dlepow/fix2

Batch: author metadata updates

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0329b

tamram committed:
adding metadata to include file

ktoliver committed:
Merge pull request #36739 from jeffgilb/330

330

GitHubber17 committed:
Merge pull request #36671 from rwike77/appsecurityupdates

moved security articles in toc

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ASG-GA

v-shils committed:
Merge pull request #36738 from roygara/Refresh

Freshness update / small bugfix for a few managed disks articles

ktoliver committed:
Merge pull request #36736 from KumudD/lbaz0330

Load balancer and Availability Zone links update

v-geberr committed:
edits

Commit:
Removed preview info from overview.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ASG-GA

v-shils committed:
Merge pull request #36746 from rolyon/rolyon-toctweaks

[Azure AD RBAC] Adjustments to TOC

billmath committed:
removing preview tags

ktoliver committed:
Merge pull request #36684 from v-nagta/fb

USER STORY 1222911 SaaS App Tutorial: Workplace by Facebook update

ktoliver committed:
Merge pull request #36725 from jimdial/ASG-GA

ASG GA

ktoliver committed:
Merge pull request #36693 from v-viinde/awsuiupdate

USER STORY 1223010 SaaS App Tutorial: Amazon Web Services (AWS) Update

Commit:
Merge pull request #36724 from paulmey/patch-1

Add australiaeast CDS cluster IP

Commit:
Merge pull request #36752 from billmath/pim2

removing preview tags

Commit:
Merge pull request #36737 from tamram/tamram-0330

fixing ZRS issues

Commit:
Merge pull request #36457 from David-Noble-at-work/cosmosdb-emulator/release/2018-03-27

Updates Local Emulator documentation for the 1.21.0.6 release publish…

vhorne committed:
Amit updates via emial 3/30

tamram committed:
updating services tag

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6514

Commit:
Updating LA default retry policy

v-geberr committed:
edits

vhorne committed:
list formatting

mimig1 committed:
Cosmos DB: Updating date to reflect recent changes

Commit:
Removed some regions from manage article.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into xregion-peering-ga

PRMerger15 committed:
Merge pull request #36759 from tfitzmac/0330link

clarified parameter

roygara committed:
Updating storage disk limits table.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Refresh

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0330-nodejs-bf-fix

bandersmsft committed:
azure policy - adding prereqs to cli and ps quickstarts

PRMerger19 committed:
Merge pull request #36760 from mimig1/patch-236

Cosmos DB: Updating date to reflect recent changes

Commit:
CSP Update

Commit:
Merge pull request #36764 from bandersmsft/master

azure policy - adding prereqs to cli and ps quickstarts

rjagiewich committed:
Merge pull request #36700 from jimdial/xregion-peering-ga

Xregion peering ga

roygara committed:
Correcting table.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Refresh

roygara committed:
Updating with acrolynx feedback.

rjagiewich committed:
Merge pull request #36670 from tamram/tamram-0329b

updating storage account limits

vhorne committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into health-stat

roygara committed:
Acrolynx feedback.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Refresh

Commit:
addressing issue 6542 updating link

PRMerger12 committed:
Merge pull request #36765 from rthorn17/management-groups

CSP Update

PRMerger12 committed:
Merge pull request #36733 from rboucher/patch-3

remove near real time and newer

gsacavdm committed:
Add variations for gov app services

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rmca14 committed:
Merge pull request #36745 from v-geberr/0330-nodejs-bf-fix

web app bot template update

gsacavdm committed:
Improve vote msg in gov-webandmobile

davidmu1 committed:
response to content study

PRMerger17 committed:
Merge pull request #36762 from roygara/Refresh

Updating list with P15 tier information

rjagiewich committed:
Merge pull request #36757 from vhorne/health-stat

Amit updates via emial 3/30

v-alje committed:
Merge pull request #36772 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #36761 from shae-hurst/sh-updateexp

Updating LA default retry policy

PRMerger19 committed:
Merge pull request #36592 from gsacavdm/patch-1

Removing bad section in gov-compute

PRMerger19 committed:
Merge pull request #36770 from gsacavdm/patch-3

Add variations for gov app services

PRMerger19 committed:
Merge pull request #36773 from davidmu1/contentstudy

response to content study

PRMerger20 committed:
Merge pull request #36769 from srrengar/workbranch2

Addressing issue 6542 updating link

rjagiewich committed:
Merge pull request #36655 from miaojiang/329edits

Modified 2 docs
